src/components/dashboard/index.tsx

```diff
@@ -2,21 +2,90 @@ import * as React from "react";
 import styles from "./dashboard.module.css"
 import { DashboardItem } from "./dashboardItem";
 import { DashboardContentsModel } from "../../interfaces/dashboardModel";
+import { DragDropContext } from 'react-beautiful-dnd';
+import { connect } from "react-redux";
+import { RootState } from "../../reducers";
+import { Dispatch, bindActionCreators } from "redux";
+import * as dashboardActions from "../../actions/dashboardActions";
 
 interface Props {
-    contents: DashboardContentsModel;
-    widgets: any;
+  contents: DashboardContentsModel;
+  widgets: any;
+  actions: any;
 }
 
 export class Dashboard extends React.Component<Props> {
-    render(){
-        const rootNode = this.props.contents.sections[this.props.contents.rootSection] || this.props.contents.widgets[this.props.contents.rootSection]
-        return (
-            <div className={styles.dashboard}>
-                <DashboardItem node={rootNode} contents={this.props.contents} widgets={this.props.widgets} />
-            </div>
-        );
+  onDragEnd = result => {
+    // Sample result format
+    // {
+    //   "draggableId" : "live-feed",
+    //   "type" : "DEFAULT",
+    //   "source" : {
+    //     "index" : 0,
+    //     "droppableId" : "left"
+    //   },
+    //   "reason" : "DROP",
+    //   "mode" : "FLUID",
+    //   "destination" : {
+    //     "droppableId" : "left",
+    //     "index" : 1
+    //   },
+    //   "combine" : null
+    // }
```

Comment:
Maybe because I don't know Drag and Droppings. But mode? What does this mean? Same with index, assuming this is same as it was above.

Reply:
I mainly added this as a sample to reference what react-beautiful-dnd returns. We aren't really using mode here so that can be ignored until we actually need to use it and index represents what position a widget is dragged into